admin/code/LeftAndMain.php

```diff
@@ -536,11 +536,10 @@ protected function init() {
 			window.ss.config = " . $this->getCombinedClientConfig() . ";
 		");
 
-		Requirements::javascript(ltrim(FRAMEWORK_ADMIN_DIR . '/client/dist/js/bundle-lib.js', '/'));
+		Requirements::javascript(FRAMEWORK_ADMIN_DIR . '/client/dist/js/vendor.js');
```

Comment:
Was wondering where that came from - I thought it might've been David misunderstanding how the framework constants work...